Documentation/prerequisites-for-building.md

```diff
@@ -1,10 +1,16 @@
 The following pre-requisites need to be installed for building the repo:
 
+# Install .NET Core 3.1 SDK
```

Comment:
.NET Core SDK is not a strict requirement to build the repo. The build script will work without it. It is convenient to have the .NET Core SDK installed to work with the repo.

Reply:
Then I do not get why it was in the "building from command line" page. It's not obvious why it is needed when running from command line.

Reply:
The page is `how-to-build-and-run-ilcompiler-in-console-shell-prompt.md`. The .NET Core SDK is not required for the build part. But it is needed by the steps that show how to run it.